zero/meta-libcsp/recipes-libcsp/libcsp_0.bb

```diff
@@ -0,0 +1,21 @@
+LICENSE = "MIT"
+LIC_FILES_CHKSUM = "file://LICENSE;md5=2915dc85ab8fd26629e560d023ef175c"
+
+SRCREV = "${AUTOREV}"
+SRCBRANCH = "develop"
+SRC_URI = "git://github.com/libcsp/libcsp.git;protocol=ssh;branch=${SRCBRANCH};"
+
+PACKAGES = "${PN} ${PN}-dbg"
+PROVIDES = "${PN} ${PN}-dbg"
```

Comment:
`PROVIDES` って`libcsp-dbg` として recipe を指定できるようになるだけよね？ debug したいときにどうやったら debug symbol が入るんだろうか？

Reply:
https://docs.yoctoproject.org/3.3/ref-manual/qa-checks.html
> - \<packagename\> rdepends on <debug_packagename> [debug-deps]
>
> A dependency exists between the specified non-dbg package (i.e. a package whose name does not end in -dbg) and a package that is a dbg package. The dbg packages contain debug symbols and are brought in using several different methods:
> 
> Using the dbg-pkgs [IMAGE_FEATURES](https://docs.yoctoproject.org/3.3/ref-manual/variables.html#term-IMAGE_FEATURES) value.
> 
> Using [IMAGE_INSTALL](https://docs.yoctoproject.org/3.3/ref-manual/variables.html#term-IMAGE_INSTALL).
> 
> As a dependency of another dbg package that was brought in using one of the above methods.
> 
> The dependency might have been automatically added because the dbg package erroneously contains files that it should not contain (e.g. a non-symlink .so file) or it might have been added manually (e.g. by adding to [RDEPENDS](https://docs.yoctoproject.org/3.3/ref-manual/variables.html#term-RDEPENDS)).

IMAGE_INSTALLかIMAGE_FEATURESを使って明示的に入れる方法があるみたいです．
逆に，dependに `libcsp-dbg`を追加するだけではだめでした

Reply:
`PROVIDES` はプロバイダ名(レシピの識別子)で，デフォルトでファイル名(${PN})となっているみたいなので，消しておきます．